README.md

```diff
@@ -97,6 +97,26 @@ defmodule MyPhoenixAppWeb.Router do
 end
 ```
 
+### Content Security Policy
+
+Content security policy nonces can be passed into the router to allow usage of strict content security policies throughout an application.
+
+This can be achieved by passing in a `csp_nonce_assign_key` to the `FunWithFlags.UI.Router` forward. Values for the nonces should be set in the conn assigns before reaching this router.
+
+The value of this can either be a single nonce assign key, or separate assign keys for script and style tags.
+
+For example:
+
+``` elixir
+forward "/", FunWithFlags.UI.Router, namespace: "feature-flags", csp_nonce_assign_key: :my_csp_nonce
+```
+
+Or:
+
+``` elixir
+forward "/", FunWithFlags.UI.Router, namespace: "feature-flags", csp_nonce_assign_key: %{style: :my_style_nonce, script: :my_script_nonce}
+```
```

Comment:
Also, I suggest to provide _full_ examples, that also show how the nonce values can be generated and set in the assigns and response headers. In other words, it's important to show how to use this end-to-end, rather than just mention it without clear instructions.

Reply:
I think the [instructions from Oban](https://getoban.pro/docs/web/2.10.5/Oban.Web.Router.html#module-content-security-policy) are pretty clear, would you be open to basically lifting this into the docs?

Reply:
The way I understand them, the instructions from Oban show how to configure the names of the assign keys for the nonces. They don't show where the nonces are supposed to come from. I suppose they would come from a different plug, but a full example would be helpful.

Reply:
Good point! I already had a nonce stored in the `conn` (originally with [put_private/3](https://hexdocs.pm/plug/Plug.Conn.html#put_private/3) and then I added it to the assigns as well for Oban). I can work on a full example.

Reply:
Funny enough, [this](https://danschultzer.com/posts/content-security-policy-with-liveview?utm_source=elixir-merge) just popped up in my feed reader. Our solution is different than that.

Reply:
Sorry for the delay, but I just wrote [2 examples](https://gist.github.com/jc00ke/9321d29acb3c09c580225d91bfa8d56a), 1 simple with a single nonce and another more advanced with nonces specific to style and script tags.